docs/conventions.md

```diff
@@ -89,6 +89,16 @@ No conventions yet
 - Contract __MUST__ exist only in case existing logic provides for an extension by the user.
 - Contract is a public API and __MUST__ follow [Spryker plugin interfaces specification](https://spryker.atlassian.net/wiki/spaces/RFC/pages/1038092073/INTEGRATED+RFC+Plugin+interface+specification).
 
+## REST API
```

Comment:
Please, move conventions to a separate PR. We need to think about it more carefully

Reply:
https://github.com/spryker-sdk/sdk/pull/247